co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureTimeInterface.java

```diff
@@ -1,69 +0,0 @@
-/*
- * Copyright (C) 2023 LEIDOS.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
```

Comment:
This class was not really necessary. It only contained really a single functionality and required the infrastructuremanager to do it. I instead moved this functionality to the infrastructuremanager